reference/settings.v3.yml

```diff
@@ -955,6 +955,7 @@ components:
         default_shopper_language:
           type: string
           default: en
+          example: 'en, ENGL'
```

Comment:
It's better to put examples with just the lang code `en` and locale lang code like `en-US`